apps/css-workshop/src/pages/button.astro

```diff
@@ -53,296 +54,288 @@ import Layout from './_layout.astro';
 
   <section id='demo-high-visibility'>
     <div id='demo-high-visibility-small' class='demo-container'>
-      <button class='iui-button' data-iui-variant='high-visibility' data-iui-size='small'>
+      <Button_ variant='high-visibility' size='small'>
         <span>Small</span>
```

Comment:
I think this `<span>` should be added within the component, rather than by the consumer

Reply:
Okay, will do.